content/guide/cli-basics.md

```diff
@@ -0,0 +1,131 @@
+---
+title: CLI Basics
+---
+
+The development workflow starts with the [NativeScript CLI](https://www.npmjs.com/package/nativescript).
+
+## CLI Basics
+
+In this article, you'll  learn the basics of the NativeScript command-line interface, including how to create new apps, how to get those apps running on devices, and how to set up a development workflow that lets you iterate fast.
+
+### Creating a new project
+
+To create a new NativeScript app project, run:
+
+```cli
+ns create
+```
+
+The NS CLI will walk you through selecting a template using interactive prompts.
+
+You can also use the `--template` flag with the `ns create` command to target a specific template
+
+```cli
+ns create HelloWorld --template @nativescript/template-hello-world-ts
+```
+`HelloWorld` above is the app name and the `--template` option tells the NativeScript CLI to scaffold an app using a predefined template named “@nativescript/template-hello-world-ts” found [here](https://github.com/NativeScript/nativescript-app-templates/tree/master/packages/template-hello-world-ts).
+
+For a full list of the templates you can use, see the [full list here](https://github.com/NativeScript/nativescript-app-templates/tree/master/packages)
+
+The `create` command will take a minute to complete, as the NativeScript CLI needs to download a few dependencies while setting up your new app.
```

Comment:
I think we don't need this here, it's been documented at:
https://beta.docs.nativescript.org/guide/creating-a-project